packages/SystemsThinkingWorkbench-Core.package/STWSimulation.class/instance/stopSimulation.st

```diff
@@ -2,4 +2,4 @@ simulation
 stopSimulation
 	
 	self setSimulationRunning: false.
-	self playButton label: 'Start'.
\ No newline at end of file
+	self playButton label: 'Play'.
```

Comment:
As wanted by the customer in the last meeting, please name the button 'Resume' instead.